app/presenters/user_analytics_presenter.rb

```diff
@@ -43,43 +34,80 @@ def display_percentage(numerator, denominator)
     "#{percentage.round(0)}%"
   end
 
+  def daily_period_list
```

Comment:
This class and these methods make a lot of sense to me 👍 The diff is ugly, but that's ok 😄